engine/src/main/java/org/hibernate/search/engine/search/highlight/HighlightConfiguration.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.highlight;
+
+public interface HighlightConfiguration {
```

Comment:
Perhaps rename this class to `SearchHighlighter` if you follow my suggestion to use `.highlighter` methods in the query DSL?